tests/testthat/test-undesirable_function_linter.R

```diff
@@ -16,3 +16,11 @@ test_that("linter returns correct linting", {
               ),
               linter)
 })
+
+test_that("it's possible to NOT lint symbols", {
+  linter <- undesirable_function_linter(
```

Comment:
i think an illustrative example to include here would be something with `lapply`